cleanlab/datalab/internal/issue_manager/knn_graph_helpers.py

```diff
@@ -0,0 +1,64 @@
+import numpy.typing as npt
+from scipy.sparse import csr_matrix
+
+
+from typing import Any, Dict, Optional, Tuple, cast
+
+from cleanlab.internal.neighbor.knn_graph import create_knn_graph_and_index
+from cleanlab.typing import Metric
+
+
+def num_neighbors_in_knn_graph(knn_graph: csr_matrix) -> int:
+    """Calculate the number of neighbors per row in a knn graph."""
+    return knn_graph.nnz // knn_graph.shape[0]
+
+
+def _process_knn_graph_from_inputs(
+    kwargs: Dict[str, Any], statistics: Dict[str, Any], k_for_recomputation: int
```

Comment:
Rename kwargs

Reply:
Renamed it to `user_find_issues_kwargs` for extra clarity. Renamed some of the variables accordingly.